src/client/stories/editor.stories.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Editor } from '../components/editor';
```

Comment:
[We have aliases for the `client` directory.](https://github.com/bocoup/threeflows/blob/teacher-moments/src/client/webpack.config.js#L27) If you wanted to, you could user it here as 
```
import { Editor } from '@client/components/editor';
```
It's not a big deal, but really convenient if we ever need to change the directory structure much or in deeply nested files.